cola_inference.py

```diff
@@ -138,6 +139,9 @@ def main(cl_arguments):
             )
             args.cuda = -1
 
+    if args.tokenizer == "auto":
```

Comment:
Is there any danger in overriding the value in args here, rather than creating a new, separate, tokenizer_name var? This seems like the simplest to maintain, but I just want to make sure there's not some assumption that I'm breaking...

Reply:
(Same Q applies to main.py)

Reply:
I'll take a more thorough look later, and although the select_tasks function looks sane, the indexer test is failing (and the point in which it's failing indicates that input_module != tokenizer_name for BERT.

Reply:
Thanks! Look through comments etc. if you have time, but it's not ready for a full review—I want to verify that most of the tasks behave as expected, which is taking a while.

Reply:
Indexer is fixed, though!